lib/components/narrative/narrative-itineraries.js

```diff
@@ -148,6 +150,32 @@ class NarrativeItineraries extends Component {
 
     if (!activeSearch) return null
 
+    // Merge duplicate itineraries together and save multiple departure times
+    let nextIndex = 0
```

Comment:
Would you be able to use the current index provided as the third argument to the reduce callback?

Reply:
This is so good I totally forgot about that argument thank you!